docs/uptime/analytics/open-web-analytics.md

```diff
@@ -0,0 +1,188 @@
+
+    - **CentOS 7**
+
+          systemctl enable mariadb
+          systemctl start mariadb
+          systemctl enable httpd
+          systemctl start httpd
+
+    - **CentOS 6**
+
+          chkconfig mysqld on
+          service mysqld start
+          chkconfig httpd on
+          service httpd start
+
+    Debian and Ubuntu users will need to restart the Apache2 daemon:
+
+        service apache2 restart
+
+## Set up MySQL
+
+1.  Run `mysql_secure_installation` to secure your database:
+
+        mysql_secure_installation
+
+    You should answer yes to most of the prompts. CentOS users, make sure to set a strong password for the root user since it is initially blank. This is the root user for `mysql` and is not related the systems root user.
+
+2.  Enter the MySQL CLI:
+ 
+        mysql -u root -p
+
+    You'll need to enter the password you set in the step above, or when you installed `mysql-server`.
+
+3.  Create a database named `owadb`: 
+
+        CREATE DATABASE owadb;
+
+4.  Create a user named `owadbuser`. Replace the example password, `owadbpassword`, with a strong password of your choice. This information will be needed later to configure OWA.
+
+        GRANT ALL PRIVILEGES ON owadb.* TO owadbuser@localhost IDENTIFIED BY 'owadbpassword';
+
+5.  Exit the MySQL CLI:
+
+        FLUSH PRIVILEGES;
+        quit
+
+## OWA
+
+### Install
+
+1.  Navigate to your document root folder for your webserver, usually `/var/www/html` by default.:
+
+        cd /var/www/html
+
+2.  Download the OWA package:
+
+        wget https://github.com/padams/Open-Web-Analytics/archive/1.5.7.tar.gz
+
+    {: .note}
+    >
+    >Version 1.5.7 is the current version and may be different by the time you read this. Please check [The Open Web Analytics](http://www.openwebanalytics.com/) site for the latest information.
+
+3.  Unpack the downloaded file:
+
+        tar xf 1.*.tar.gz
+
+4.  Change ownership of the `owa` folder to the Apache daemon user:
+
+    - **CentOS**
+
+          chown -R apache:apache Open*
+
+    - **Debian & Ubuntu**
+
+          chown -R www-data:www-data Open*
+
+5.  **Recommended:** Rename the OWA folder:
+
+        mv Open-Web-Analytics-1.5.7 owa
+
+6.  Delete the tar file:
+
+        rm -rf 1.*.tar.gz
+
+### Configure
+
+1.  Navigate to the OWA installation page in your webbrowser. Replace `your.domain` with your Linode's IP address or FQDN:
+
+        http://your.domain/owa/
+
+
+
+2.  After clicking on **Let's Get Started**, you should see a configuration page for your OWA installation. The first field will be filled in by OWA with the path to your OWA installation. The other fields on the page should be filled in with the information you set in the MySQL CLI:
+ 
+    [![Open Web Analytics set up screen.](/docs/assets/owa-install_small.png)](/docs/assets/owa-install.png)
+
+3.  Click **Continue...**. 
+
+4.  Create a user account and define a domain to track. This user account will be how you log in to see your OWA statistics.
```

Comment:
I made this note on what is now an outdated diff, but it still applies:

During my tech testing it appears that this step in the installation page also has us create the first site to track, so that when we log in we already have some code presented. This affects both this step and the next section.